lib/composer.json

```
@@ -6,7 +6,8 @@
     "license": "AGPL-3.0",
     "require": {
         "php": ">=7.0.0",
-        "phpmailer/phpmailer": "6.6.0"
+        "phpmailer/phpmailer": "6.6.0",
+        "phpgt/cssxpath": "v1.1.4"
```

```suggestion
        "phpgt/cssxpath": "v1.1.4",
        "phpmailer/phpmailer": "6.6.0"
```

Just keeping alphabetic order :-)

---

👍